tinyxml2.cpp

```diff
@@ -802,6 +802,8 @@ void XMLNode::Unlink( XMLNode* child )
         child->_next->_prev = child->_prev;
     }
 	child->_parent = 0;
+	child->_next = 0;

```

Comment:
Isn't very important but won't hurt to move each `=0` inside corresponding `if( !=0)` above.